.vscode/settings.json

```diff
@@ -0,0 +1,2 @@
+{
```

Comment:
please add `.vscode` to the `.gitignore` file in order to ignore prevent pushing files like this one to the source